### DISCUSSION TITLE
Solution for "SDXL - model not loaded error"

### DISCUSSION THREAD

**Discussion by cgidesign-de** [vladmandic/sdnext#1691] [Q&A]
Thanks for implementing SDXL.

One issue I had, was loading the models from huggingface with Automatic set to default setings.
It has "fp16" in "specify model variant" by default.
Once downloaded, the models had "fp16" in the filename as well. But Automatic wants those models without fp16 in the filename. I just went through all folders and removed fp16 from the filenames - since then it is working.

![automatic](https://github.com/vladmandic/automatic/assets/115958660/a7a8b71b-66ed-4a3a-bf14-bfa16e94a2e1)

**Comment by StudioDUzes** [comment]
Thanks for the advice, it works! but now: RuntimeError: The GPU device does not support Double (Float64) operations!
What graphics card do you have?

**Comment by cgidesign-de** [comment]
I am running a Nvidida RTX 3090.
Not sure about your error - maybe check in settings if FP32 is set instead of FP16 (but this is pure guessing on my side).
